057-deprecate-and-remove-envvar-config-provider.md

```diff
@@ -0,0 +1,47 @@
+# Deprecate and remove EnvVarConfigProvider
+
+This proposes deprecating the [kafka-env-var-config-provider](https://github.com/strimzi/kafka-env-var-config-provider) project. It also proposes a timeline to archive the project and stop including the JAR in Strimzi releases.
+
+## Current situation
+
+In [proposal 30](https://github.com/strimzi/proposals/blob/main/030-env-var-config-provider.md), we added an implementation of the [ConfigProvider](https://kafka.apache.org/35/javadoc/org/apache/kafka/common/config/provider/ConfigProvider.html) interface of Apache Kafka that allows retrieving configuration values at runtime from environment variables. This implementation lives in the `kafka-env-var-config-provider` project. This is especially useful to safely configure sensitive settings in virtualized/containerized environments like Kubernetes.
+
+For example, a Kafka client can use this provider with specified environment variables:
+```properties
+config.providers=env
+config.providers.env.class=io.strimzi.kafka.EnvVarConfigProvider
+option1=${env::FIRST_ENV_VAR}
```

Comment:
I think here should be only one `:` as we have it in the motivation section? :)
```suggestion
option1=${env:FIRST_ENV_VAR}
```

Reply:
Good catch!